src/ooh_wee_task_launcher.erl

```diff
@@ -0,0 +1,75 @@
+-module(ooh_wee_task_launcher).
```

Comment:
This module listens on /ooh_wee/tasks/$TASK_GROUP/ for new tasks, then creates a new instance of the module which is associated with this task group and triggers the work.

The good thing is, that only _one_ instance of this will be triggered at any point in time. With careful use of the ZooKeeper API we can give a 99% guarantee of "at most once" execution.